src/flask/helpers.py

```diff
@@ -841,38 +845,37 @@ def _find_package_path(root_mod_name):
         if spec.origin in {"namespace", None}:
             return os.path.dirname(next(iter(spec.submodule_search_locations)))
         # a package (with __init__.py)
-        elif spec.submodule_search_locations:
+        if spec.submodule_search_locations:
             return os.path.dirname(os.path.dirname(spec.origin))
         # just a normal module
-        else:
-            return os.path.dirname(spec.origin)
+        return os.path.dirname(spec.origin)
```

Comment:
With the old code every return was on the same levl - now the last one is on a different level. Which option to choose is completely personal preference though, but I don't think sending PRs to projects you are not related to do make changes that are purely based on personal preference is helpful.